app/controllers/financialOverview/psa/AllPenaltiesAndChargesController.scala

```diff
@@ -60,13 +60,13 @@ class AllPenaltiesAndChargesController @Inject()(
 
       psaPenaltiesAndChargesService.getPenaltiesForJourney(request.idOrException, journeyType).flatMap { penaltiesCache =>
 
-        val filteredPenalties: Seq[PsaFSDetail] = penaltiesCache.penalties
+        val filteredPenalties: collection.Seq[PsaFSDetail] = penaltiesCache.penalties
```

Comment:
Would be more concise to import this package

Reply:
Amended